UNRELEASED.md

```diff
@@ -5,6 +5,8 @@
 **Added:**
   - `BpkCalendar`
     - New functionality – labels for each cell.
+  - `BpkTextField`
```

Comment:
I'm not sure whether we should put component updates in "Added" or "Fixed" section. Alternatively, we can create "Updated" section.

Reply:
I think `Updated` would be a good title 👍 
Did we decide this is visually breaking or not?

Reply:
I think we need to release it as a breaking change.